system/lib/libc/musl/src/internal/libc.h

```diff
@@ -5,31 +5,35 @@
 #include <stdio.h>
 #include <limits.h>
 
+struct __locale_map;
+
+struct __locale_struct {
+	const struct __locale_map *volatile cat[6];
+};
+
+struct tls_module {
+	struct tls_module *next;
+	void *image;
+	size_t len, size, align, offset;
+};
+
 struct __libc {
-	void *main_thread;
+	int can_do_threads;
```

Comment:
On further inspection, not sure if that's worth doing. Currently, and when the field was added (https://github.com/bpowers/musl/commit/dab441aea240f3b7c18a26d2ef51979ea36c301c), it only serves to return `ENOSYS` from `pthread_create`, which gets provided by library JS code.